code/dqe/symfilecheck.q

```diff
@@ -0,0 +1,8 @@
+\d .dqe
+hdbdir:@[value;`hdbdir;`hdb]
+symfilecheck:{[directory;filename]                     // if sym file exists
+  fullpath:.Q.dd[directory]filename;
+  $[not ()~key hsym fullpath;
```

Comment:
I think you can use `.os.Fex` here https://github.com/AquaQAnalytics/TorQ/blob/master/code/common/os.q#L4

```suggestion
  $[.os.Fex fullpath;
```

(could probably also lose the `fullpath` intermediate variable)